files-community/Files
# Code Quality: Introduced IStorageSecurityService

### Summary

- Removal of Vanara reference (4 out of 37 files):
  - [x] SecurityViewModel
  - [x] SecurityAdvancedViewModel
  - [x] FileSecurityHelpers (to IStorageSecurityService)
  - [x] AccessControlPrincipal
- Removal of Utils namespace (9 out of 132 files):
  - [x] Utils/Storage/Security/*
  - [x] Utils/Storage/Helpers/FileSecurityHelpers

### Resolved / Related Issues

- https://github.com/files-community/Files/issues/14572
- https://github.com/files-community/Files/issues/15000

### Steps used to test these changes

1. Open Files
2. Go to security and advanced security pages
3. Confirm no crashes and problems occur
4. Add a new entry
5. Delete the entry
